tests/test_facebook_attribution_window.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+from tap_tester import runner, connections
+
+from base import FacebookBaseTest
+
+class FacebookAttributionWindow(FacebookBaseTest):
+
+    # set attribution window
+    attrubution_window = 7
```

Comment:
Define constants in caps like ATTRIBUTION_WINDOW

Reply:
Added